src/WebJobs.Extensions.DurableTask/ContextInterfaces/IDurableOrchestrationClient.cs

```diff
@@ -52,6 +54,7 @@ public interface IDurableOrchestrationClient
         /// <param name="instanceId">The ID of the orchestration instance to check.</param>
         /// <returns>An HTTP 202 response with a Location header and a payload containing instance control URLs.</returns>
         IActionResult CreateCheckStatusResponse(HttpRequest request, string instanceId);
+#endif
```

Comment:
If we're upgrading our full framework target to .NET Framework 4.7.1, and it supports .NET Standard 2.0, shouldn't we also be able to use these types/methods in Functions v1?